logs_http.module

```diff
@@ -93,146 +68,27 @@ function _logs_http_decode_exception($exception) {
  * devel_boot() registers this function as a shutdown function.
  */
 function logs_http_shutdown() {
-  if (!$events = logs_http_get_registered_events()) {
+  if (!LogsHttpLogger::isEnabled()) {
```

Comment:
I think `LogsHttpLogger` should be a service, so the logic we have here can be swapped out. This will also move all `LogsHttpLogger`'s static methods to be non-static ones.